src/QueryPipeline/QueryPipelineBuilder.cpp

```diff
@@ -370,7 +370,6 @@ std::unique_ptr<QueryPipelineBuilder> QueryPipelineBuilder::joinPipelines(
             return processors;
         };
         right->transform(concurrent_right_filling_transform);
-        right->resize(1);
```

Comment:
So, is this comment irrelevant?
https://github.com/ClickHouse/ClickHouse/pull/36415#discussion_r854709061